modules/storage-multi-attach-error.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-storage-issues.adoc
+
+[id="storage-multi-attach-error_{context}"]
+= Resolving multi-attach errors
+
+.Problem
+When a node crashes or shuts down abruptly, the ReadWriteOnce (RWO) volume is expected to be unmounted from the node so that it can be used by a Pod scheduled on another node.
+
+Mounting on a new node is not possible, however, because the failed node was unable to properly unmount the attached volume.
```

Comment:
Just so that I understand - Is mounting on a new node always not possible? Or only in the cases where the failed node was unable to unmount the attached volume?

If the latter, then clarification is needed. If the former, then I might suggest moving "however" to the front instead of in the middle of the sentence. wdyt?

```
However, mounting on a new node is not possible because the failed node was unable to properly unmount the attached volume.
```

Reply:
The former is correct; for RWO volumes, if the node fails, the RWO volume can't mount to the new node because the failed node doesn't unmount it. I agree with moving "however" to the front. I also reworded the sentence for clarity. 

Great suggestion, thanks!